manifests/db.pp

```diff
@@ -23,7 +23,7 @@
     charset  => $charset,
     collate  => $collate,
     provider => 'mysql',
-    require  => [ Class['mysql::server'], Class['mysql::client'] ],
+    require  => [ Class['mysql::client'] ],
```

Comment:
I would like to see here 
<pre><code>
require  => defined(Class['mysql::server'])? {
                   true  => [ Class['mysql::server'], Class['mysql::client'] ],
                   false => [ Class['mysql::client'] ]
                },</code></pre>